src/api/polkadot/polkadotContext.ts

```diff
@@ -84,8 +84,8 @@ export const providePolkadotContainer = (initApi: ApiPromise) => {
 
 export const usePolkadotContainerContext = () => {
     // allow access to the readonly state provided by the container
-    return {
+    return toRefs({
```

Comment:
```
toRefs() expects a reactive object but received a plain one.
```
This object should be reactive before it can use `toRefs()`. The error seems to appear in a couple of places.